src/main/java/net/minestom/server/command/builder/Arguments.java

```diff
@@ -49,6 +49,10 @@
         });
     }
 
+    public boolean has(@NotNull Argument<T> argument) {
```

Comment:
You should probably use the wildcard `?` instead (it doesn't compile right now)

Reply:
Yes, i have failed this ><